### DISCUSSION TITLE
Only small fraction of genes have predicted eRegulons

### DISCUSSION THREAD

**Discussion by SidG13** [aertslab/scenicplus#156] [Q&A]
Hi all, I'm running a merged multiome sample (removing a batch effect) with ~17k cells. My GTF has ~18k genes, yet I only have about 2k genes which have predicted eRegulons. I realize this is a broad question, but is this expected?

I realize that if a gene isn't expressed in too many cells, then perhaps it won't have predictions. But what I found odd was that when I ran with just a single sample (~3k cells), I had eRegulon predictions for some genes that were then lost when I ran the larger merged dataset. This doesn't seem to be normal behaviour, but I'm not too sure. 

All the steps in the pipeline seemed to run fine, without problems so it's likely an issue with my data. I'm wondering if there are steps to take to loosen some statistical restrictions which might be causing this sort of dropout?

Thanks!

**Comment by janursa** [comment]
i am not the maintainer but have you checked if this is because of filtering posed on highly variable genes?
